test/test-server/ccloud_handlers.go

```diff
@@ -61,7 +61,6 @@ const (
 	serviceAccountID           = int32(12345)
 	serviceAccountResourceID   = "sa-12345"
 	identityProviderResourceID = "op-12345"
-	identityPoolResourceID     = "pool-12345"
```

Comment:
i see it's still being used in multiple places. what was the intention here to delete it？

Reply:
I removed it because the variable itself was only used in once place after I replaced another instance of it.

But yeah, I just did a search and the string `"pool-12345"` looks like it appears in more places, so I can restore this and switch the other uses of this string to this variable.